src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -493,7 +496,8 @@ internal static (List<PackageArtifactModel>, List<BlobArtifactModel>) GetPackage
                 {
                     Attributes = new Dictionary<string, string>
                     {
-                        { "NonShipping", package.NonShipping.ToString().ToLower() },
+                        { NonShippingAttributeName, package.NonShipping.ToString().ToLower() },
+                        { DotNetReleaseShippingAttributeName, package.DotNetReleaseShipping.ToString().ToLower() }
```

Comment:
Has this property already been added to AssetManifests in arcade?

Reply:
yes, but this should work also if the prop is not set in the asset manifest

Reply:
Do we know how the xml parser works? Does it set the property to false if it's not found? or is it null?

Reply:
looks like it will bind to the `default` value if it's missing in the xml, so we're good

Reply:
yeah I tried it out